### ISSUE TITLE
GeocodeResponse score

### ISSUE THREAD

**Issue by joshlk** [foursquare/twofishes#89]
Hi, I've got a couple of related questions:

a) When debugging is turned on I noticed that each possible location is given a 'score' which is then used to choice which location to pick. Could this score be used as a measure of confidence i.e. how confident the final result is in-fact the place in the query? This would allow me to compare different queries and state which one I am more confident about. 

b) Is there an option which includes the score in a `GeocodeResponse`. In the Thrift I can see the score is optionally included but even if I ask for everything the fields are left blank.

Thanks in advanced

**Comment by rahulpratapm** [comment]
The scores are somewhat arbitrary at this point. Their values are only meant to be used for comparison for ranking--they don't really mean anything in absolute terms. We're thinking about introducing a notion of confidence that is informed by how good the match is but we're a long way away from shipping that.

It appears we deprecated the `scoringFeatures` field on `GeocodeInterpretation` and added `scores`at some point in the past before I started working on twofishes:

```
5: optional ScoringFeatures scoringFeatures_DEPRECATED
6: optional InterpretationScoringFeatures scores
```

As InterpretationScoringFeatures is currently defined, it only returns coverage info for reverse geocode queries with a radius (or non-point geometry in general). We could potentially add fields to it that make it meaningful for forward geocoding as well.
